samples/modules/nanopb/CMakeLists.txt

```diff
@@ -5,6 +5,7 @@ cmake_minimum_required(VERSION 3.20.0)
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 project(nanopb_sample)
 
+set(NANOPB_OPTIONS "-I${CMAKE_SOURCE_DIR}")
```

Comment:
curious.

You add `CMAKE_SOURCE_DIR` == `samples/modules/nanopb`, but you add the file: `simple.options` to this folder: `samples/modules/nanopb/src`.

If that is correct, please comment this code as to why `src` is not needed with `NANOPB_OPTIONS`.

Reply:
It's because we're using `RELPATH .` just under it. I've added the comment.

Reply:
ahh, so the `NANOPB_OPTIONS` is not directly coupled to the `simple.options` file, is that correct ?

Guess the reason I got confused is because the commit message talks about:
>  Add .options file to the sample 

which is done in `nanopb/src` and then there was this additional line, thus I thought they were related.